tests/api_workflow/test_api_workflow_upload_embeddings.py

```diff
@@ -88,30 +88,21 @@ def test_upload_comma_filenames(self):
                         special_char_in_first_filename=invalid_char)
 
     def test_set_embedding_id_success(self):
-        embedding_name = self.api_workflow_client.embeddings_api.embeddings[0].name
+        embedding_name = self.api_workflow_client._embeddings_api.embeddings[0].name
         self.api_workflow_client.set_embedding_id_by_name(embedding_name)
 
     def test_set_embedding_id_failure(self):
         embedding_name = "blibblabblub"
         with self.assertRaises(ValueError):
             self.api_workflow_client.set_embedding_id_by_name(embedding_name)
 
-    def test_upload_existing_embedding(self):
```

Comment:
Removed, because already covered by the following tests (testing with appending).